components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -8224,6 +8231,11 @@ public static Cache getCache(final String cacheManagerName, final String cacheNa
         return Caching.getCacheManager(cacheManagerName).getCache(cacheName);
     }
 
+    private static String getCacheName(String cacheName) {
+        return Boolean.parseBoolean(ServerConfiguration.getInstance().getFirstProperty("Cache.ForceLocalCache"))
```

Comment:
use braces to improve readability

return (Boolean.parseBoolean(ServerConfiguration.getInstance().getFirstProperty("Cache.ForceLocalCache"))
                && !cacheName.startsWith("$__local__$.")) ? "$__local__$." + cacheName : cacheName;

Reply:
resolved by #8332